### PR TITLE
feat(migration): add questions commented by user controller with integration tests

### LINKED ISSUE #98: [Profile] Migrate controller to get list of questions commented by user
Migrate controller worked on this ticket https://github.com/wizeline/wize-q-back/issues/500

### PR DESCRIPTION
**Pull request by JerryArram**
#### What does this PR do?
Adds questions commented controller and the necessary tests. 

#### Where should the reviewer start?
app/controllers/profile/questionsCommented.js
tests/integration/controllers/profile/questionCommented.test.js

#### How should this be manually tested?
1. Run the command 'npm run test:integration' in the console and make sure all the test are passed.

#### What are the relevant tickets?
Closes #98 

#### Screenshots
:camera:
Screen Shot 2022-11-28 at 1 11 51 p m

### CODE REVIEW

File: app/controllers/profile/questionsCommented.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                where:{
+                 question_id: {
+                     in: values,
+                    }, 
+             },
+            });
+    
+         return{
+                success:'Questions commented by user obtained succesfully',
+                getQuestionCommented
```
Comment (albertcastaned) [inline app/controllers/profile/questionsCommented.js:38]: The return value could have a clearer name like "questions" maybe?

File: app/controllers/profile/questionsCommented.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            } 
+    }
+} catch(error) {
+    return {
+        error: {
+            message: 'The user was not found',
+            detail: 'The user was not found',
```
Comment (albertcastaned) [inline app/controllers/profile/questionsCommented.js:45]: Could you add the userEmail on the detail message? So that when debugging it is easier

**Review by albertcastaned — COMMENTED** [review]
Left some suggestions @JerryArram

File: app/controllers/profile/questionsCommented.js (hunk 2 above)
Comment (JerryArram) [reply app/controllers/profile/questionsCommented.js]: added all suggestions

**Review by albertcastaned — APPROVED** [review]
LGTM!

**Pull request merged**